lib/message.rb

```diff
@@ -0,0 +1,33 @@
+require 'pg'
+
+class Message
+  # def initialize
+  #   @messages = []
+  # end
+
+  # def add(message)
+  #   @messages << message
+  # end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+    result = connection.exec('SELECT * FROM peeps;')
+    #p result
+    result.map { |peep| peep['message'] }
+    # return @messages
+  end
+
+  def self.create(message:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
```

Comment:
You're using this same piece of code in more than one place in this file. How might you get rid of this duplication?